### ISSUE TITLE
Increase size of share dialog

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/desktop#4134]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### problem
Share dialog is too small for most information to fit in:

<img width="252" alt="Screenshot_20220104_112817" src="https://user-images.githubusercontent.com/551757/148064598-3e03ce33-9a2c-4330-bb5f-ad42a1bd7721.png">
<img width="235" alt="Screenshot_20220104_112740" src="https://user-images.githubusercontent.com/551757/148064601-695f556a-2608-47bf-855a-08efcd03b1ca.png">
<img width="235" alt="Screenshot_20220104_112718" src="https://user-images.githubusercontent.com/551757/148064602-372af9ce-fff9-4728-ae89-82a78777cbd2.png">

### What should happen
Dialog should be... larger ;-)

It doesn't have to be huge, but why this tiny? making it twice as big would still be small compared to most dialogs on the desktop.

### Client configuration
Client version: 3.4.1 Linux

**Comment by jospoortvliet** [comment]
@jancborchardt what do you think? Making the dialog wider would make it possible to read the texts we put in there, like in the share field & drop-down menus...

**Comment by allexzander** [comment]
@camilasan You might have an opinion since you've been working on adding a vertical scrollbar to it recently :)

**Comment by jancborchardt** [comment]
The width of the sidebar on web is 310px, so if this dialog is smaller than that, we could adjust it to this size minimum/default. And then as height something like 350px?

Regarding this screenshot, something that would also alleviate the width issues is if we can do this similar to the web app where we use bigger entries with sublines for the emails (and actual avatars), if possible?
![image](https://user-images.githubusercontent.com/925062/148218295-6f57baa5-91bb-43af-bf16-9c7fdacc20fb.png)

**Comment by jospoortvliet** [comment]
> The width of the sidebar on web is 310px, so if this dialog is smaller than that, we could adjust it to this size minimum/default. And then as height something like 350px?

So it's tiny compared to at least what my browser shows:

<img width="340" alt="Screenshot_20220107_155108" src="https://user-images.githubusercontent.com/551757/148561253-8b2599c6-d18a-44e5-9c05-1154dc018c37.png">

And that is at 100% zoom, I normally work at 150% on cloud.nextcloud.com:
<img width="489" alt="Screenshot_20220107_155231" src="https://user-images.githubusercontent.com/551757/148561392-8dd37fd0-b809-4e21-b3e6-8a70687f3a3a.png">

Even then, why is the width of the web UI that relevant, we don't follow the height either - screen-high would look ridiculous.

Can't we focus on making it comfortable for the user instead? At least making it 50% bigger I'd say, so the text is still readable even if a user has a larger font configured (right now, the dialog becomes quickly unreadable). I know a browser adjusts to that but our app doesn't.

Other share dialogs are far bigger, too:

https://www.howtogeek.com/wp-content/uploads/2010/04/sshot337.png
https://sylbek.de/img/onedrive-sharepoint.png

**Comment by jancborchardt** [comment]
I'm fine making it 50% wider. And height wouldnt be screen height but a bit more than width so it doesnt look off. I proposed specific values above.
